### ISSUE TITLE
Command line parameter "--syslog-facility" does not accept LOCAL0..LOCAL7

### ISSUE THREAD

**Issue by pjakuszew** [rclone/rclone#4061]
#### What is the problem you are having with rclone?

I cannot use `--syslog-facility LOCAL3` - it returns following error:

```
2020/03/12 12:26:09 Unknown syslog facility "LOCAL3" - man syslog for list
``` 

Other facilities like KERN or DAEMON (default one) are accepted.

#### What is your rclone version (output from `rclone version`)

v1.51.0

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Debian GNU/Linux 10 (buster) amd64

####  Which cloud storage system are you using? (eg Google Drive)

N/A

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

`rclone --syslog --syslog-facility LOCAL3 copy ...`

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

```
# rclone -vv --syslog --syslog-facility LOCAL3
2020/03/12 12:31:44 Unknown syslog facility "LOCAL3" - man syslog for list
```

**Comment by darthShadow** [comment]
This should be simple enough if you would like to send a PR for it. Use the golang [doc](https://golang.org/pkg/log/syslog/#Priority) as a reference and add the respective mappings in [syslog_unix.go](https://github.com/rclone/rclone/blob/ae340cf7d93fe394deb6b723805cbd2259c5bce3/fs/log/syslog_unix.go#L17)

@ncw Correct me if I am wrong and something additional is required too.

**Comment by pjakuszew** [comment]
Thank you for response, @darthShadow ! 

Indeed, it seems to be a really simple change - I have made a PR, I checked it locally and `--syslog-facility LOCAL0...7` now works correctly.

**Comment by pjakuszew** [comment]
Actually, I will cancel the PR for now - this comment in golang reference documentation worries me:

```
// These are the same up to LOG_FTP on Linux, BSD, and OS X.
```

While it did work on my environment (Debian), this comment suggests that it may not behave correctly on other Syslog implementations.

**Comment by pjakuszew** [comment]
So, standard described in https://tools.ietf.org/html/rfc5424 specifically mentions `local0..7` facilities, with following comment:

>Facility and Severity values are not normative but often used.  They are described in the following tables for purely informational purposes.

Should we just assume that we are running on OS with syslog daemon which implements the standard from RFC?

**Comment by ncw** [comment]
I would say that if it compiles then the symbols exist so it is fine.

Did the CI run clean? In particular the compile all build?

**Comment by pjakuszew** [comment]
@ncw Yes, according to CI, everything is fine: https://github.com/rclone/rclone/pull/4063/checks?check_run_id=503871184

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.52 - thank you @pjakuszew
